packages/backend/package.json

```diff
@@ -82,9 +74,7 @@
     "objection": "^3.1.1",
     "objection-db-errors": "^1.1.2",
     "oer-utils": "5.1.3-alpha.2",
-    "pg": "^8.11.2",
     "pino": "^8.15.0",
-    "pino-pretty": "^10.2.0",
```

Comment:
I believe this is still being used here: https://github.com/interledger/rafiki/blob/main/packages/backend/src/tests/app.ts#L43